app/templates/settings.hbs

```diff
@@ -45,7 +45,12 @@
   <section class="settings-section">
     <h2 class="small-title">Environment Variables</h2>
 
-    <p>Notice that the values are not escaped when your builds are executed. Special characters (for bash) should be escaped accordingly.</p>
+    <p>
+      Customize your build using environment variables. For secure tips on generating private keys
+      <a href="https://docs.travis-ci.com/user/best-practices-security#recommendations-on-how-to-avoid-leaking-secrets-to-build-logs">
```

Comment:
This link should open in a new tab. Anyways, there is an [`external-link-to`](https://github.com/travis-ci/travis-web/blob/master/app/components/external-link-to.js) component that should be used for external links. It internally bounds all necessary attributes and takes care of some security issues. 

Also we strive to keep all our external URLs in config section: https://github.com/travis-ci/travis-web/blob/master/config/environment.js#L39-L62 - you can add the URL there and then use it here with a `config-get` helper:

```
{{#external-link-to href=(config-get 'urls.urlKey')}}
  read our documentation
{{/external-link-to}}
```